src/Controllers/StandortController.cs

```diff
@@ -49,4 +49,50 @@ public StandortController(EwsContext context)
 
         return await standorte.AsNoTracking().ToListAsync().ConfigureAwait(false);
     }
+
+    [HttpPost]
+    public IActionResult Create(Standort standort)
+    {
+        context.Standorte.Add(standort);
+        context.SaveChanges();
+        return CreatedAtAction(nameof(Standort), standort);
+    }
+
+    [HttpPut]
+    public IActionResult Edit(Standort standort)
+    {
+        var standortToEdit = context.Standorte.SingleOrDefault(x => x.Id == standort.Id);
+        if (standortToEdit == null)
+        {
+            return new NotFoundResult();
+        }
+        else
+        {
+            standortToEdit.Bezeichnung = standort.Bezeichnung;
+            standortToEdit.Bemerkung = standort.Bemerkung;
+            standortToEdit.Gemeinde = standort.Gemeinde;
+            standortToEdit.GrundbuchNr = standort.GrundbuchNr;
+            standortToEdit.Bohrungen = standort.Bohrungen;
+            standortToEdit.Mutationsdatum = DateTime.UtcNow;
+            standortToEdit.UserMutation = standort.UserMutation;
+            context.SaveChanges();
+            return Ok();
+        }
+    }
+
+    [HttpDelete]
+    public IActionResult Delete(int id)
+    {
+        var standortToDelete = context.Standorte.SingleOrDefault(x => x.Id == id);
+        if (standortToDelete == null)
+        {
+            return new NotFoundResult();
```

Comment:
PP: Ich denke das kann vereinfacht werden zu:
```suggestion
            return NotFound();
```